CHANGELOG.md

```diff
@@ -5,8 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## 0.1.1 - Unreleased
+
+### Added
+
+- Handle `Opcode::GetEffectName` to resolve name display issues on some hosts ([#89]) 
+
+### Changed
+
+- Set params to be automatable by default ([#99])
+- Updated README.md to reflect org and chat changes ([#90], [#94])
```

Comment:
I am wondering if shouldn't it make more explicit that the backward compatibility has been broken because how the parameters are being handled now.

Reply:
Yes, that should be the most prominent item in the 0.2.0 changelog entry.